includes/admin/reporting/reports.php

```diff
@@ -485,18 +485,14 @@ function edd_register_downloads_report( $reports ) {
 		// Mock downloads and prices in case they cannot be found later.
 		$download       = edd_get_download();
 		$prices         = array();
-
+		$download_label = '';
 		if ( $download_data ) {
 			$download = edd_get_download( $download_data['download_id'] );
-			$prices   = $download->get_prices();
-
 			if ( $download_data['price_id'] ) {
-				$prices = array_values( wp_filter_object_list( $download->get_prices(), array( 'index' => absint( $download_data['price_id'] ) ) ) );
-
-				$download_label = esc_html( ' (' . $download->post_title . ': ' . $prices[0]['name'] . ')' );
-			} else {
-				$download_label = esc_html( ' (' . $download->post_title . ')' );
+				$args                  = array( 'price_id' => $download_data['price_id'] );
+				$download->post_title .= ': ' . edd_get_price_name( $download->ID, $args );
```

Comment:
`edd_get_price_name()` could return `false` so we should probably be checking if the return value isn't empty before appending it.